src/modules/EdgeDetect/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Detect Edges",
+    "description": "Detects the edges in an image and outputs a a greyscale image with the canny method",
```

Comment:
Please fix the double "a" in the description field's value

Reply:
Again, I apologize for being that picky!

Reply:
@ccpandhare no apologies needed in fact thank you for being so observant😁